tests/migration/eviction_strategy.go

```diff
@@ -72,7 +73,7 @@ var _ = SIGMigrationDescribe("Live Migration", func() {
 				vmi = alpineVMIWithEvictionStrategy()
 			})
 
-			It("[test_id:3242]should block the eviction api and migrate", func() {
+			It("[test_id:3242]should block the eviction api and migrate", decorators.Conformance, func() {
```

Comment:
While to some obvious, please reason in the commit message why you choose this test.

Reply:
Done